.bumpversion.cfg

```diff
@@ -1,6 +1,6 @@
 [bumpversion]
 current_version = 1.7.1
-files = setup.py docs/conf.py
+files =
```

Comment:
I'm not familiar with bumpversion, but this being empty looks odd. It's this correct?

Reply:
Good point, I just double-checked and this config option is basically not needed anymore since we're swtiching to [setuptools-scm](https://pypi.org/project/setuptools-scm/) for the version handling during package building and that simply reads the version from the git tag. We can still use bumpversion (I've tried it) and rely on its ability to bump the version up  and create a tag. In that sense it's not doing much, but I'd rather keep the config in place if the original author like to use it.

Reply:
bumpversion is (was) used primarily to update the version number in those files. With that gone, there’s little use to keep bumpversion around IMO. Although I’m unfamiliar with setuptools-scm,  I’m ok with removing the config as it fits better with Jazzband shared project setup.